saidump/saidump.cpp

```diff
@@ -150,6 +160,226 @@ void print_attributes(size_t indent, const TableMap& map)
     }
 }
 
+#define SAI_OBJECT_TYPE_PREFIX_LEN 16
```

Comment:
>16 [](start = 35, length = 2)

Can we get the number programmatically?

Reply:
lenght is "SAI_OBJECT_TYPE_" for this how to get size of it ? sizeof("SAI_OBJECT_TYPE_") -1 ?